assets/src/stories-editor/components/block-mover/draggable.js

```diff
@@ -37,6 +40,20 @@ class Draggable extends Component {
 		this.isChromeAndHasIframes = false;
 	}
 
+	/**
+	 * In the image block, ensure that the preview image itself isn't draggable.
+	 *
+	 * When the image is draggable, it engages the DropZone of the Image block,
+	 * not the DropZone that allows dragging the block on the page.
+	 * There looks to be an easier way to do this with CSS,
+	 * but -moz-user-drag: none; didn't work on Firefox.
+	 */
+	componentDidMount() {
+		document.querySelectorAll( '.block-editor-block-list__block[data-type="core/image"] img' ).forEach( ( image ) => {
+			image.setAttribute( 'draggable', 'false' );
```

Comment:
Not critical but I wonder if there's a better place for doing this -- it seems to do the same for every `Draggable` and each time for all the images.